<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemMonitor</class>
 <widget class="QMainWindow" name="SystemMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemMonitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSensores">
       <attribute name="title">
        <string>Sensores</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeSensores"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProcesos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Procesos</string>
       </attribute>
       <widget class="QTableWidget" name="tableProcesos">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>501</width>
          <height>331</height>
         </rect>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CMDLINE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NAME</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STATE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>THREADS</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="killButton">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>100</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Kill Proccess</string>
        </property>
       </widget>
       <widget class="QPushButton" name="restartButton">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>190</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Restart Machine</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabRed">
       <attribute name="title">
        <string>Red</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelRed">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHardware">
       <attribute name="title">
        <string>Hardware</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeView" name="treeHardware"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
